src/vs/workbench/contrib/terminal/browser/media/shellIntegration.ps1

```diff
@@ -16,30 +16,41 @@ if ($ExecutionContext.SessionState.LanguageMode -ne "FullLanguage") {
 $Global:__VSCodeOriginalPrompt = $function:Prompt
 
 $Global:__LastHistoryId = -1
+function Set-Serialized
```

Comment:
`Serialize-Property`? I don't think we need to use an approved verb as it's not exposed?